### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,59 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.1.0 - Obelix [xxxx-xx-xx]
```
Comment (ramprasadn) [inline CHANGELOG.md:6]: Leaving this note for myself so I don't forget to update the date before merging this PR

Comment (maxulysse) [reply CHANGELOG.md:6]: Loving the Asterix themed codename for the the release

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ custom_logo_url: https://github.com/nf-core/raredisease/
 custom_logo_title: "nf-core/raredisease"
 
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/raredisease/releases/tag/2.0.1" target="_blank">nf-core/raredisease</a>
+  This report has been generated by the <a href="https://github.com/nf-core/raredisease/tree/dev" target="_blank">nf-core/raredisease</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/raredisease/2.0.1/docs/output" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/raredisease/dev/docs/output" target="_blank">documentation</a>.
```
Comment (maxulysse) [inline assets/multiqc_config.yml:8]: if it's a release, it shouldn't point to dev.
Did you use `nf-core bump-version 2.1.0`?

File: conf/base.config [hunk 3]
```diff
@@ -57,7 +57,9 @@ process {
         errorStrategy = 'retry'
         maxRetries    = 2
     }
-    withName:CUSTOM_DUMPSOFTWAREVERSIONS {
-        cache = false
+    withName: 'NFCORE_RAREDISEASE:RAREDISEASE:.*' {
+        publishDir = [
+            enabled: false
```
Comment (maxulysse) [inline conf/base.config:62]: that's a bit extreme, no?

Comment (maxulysse) [reply conf/base.config:62]: nevermind, that's a super good idea actually

File: nextflow.config [hunk 4]
```diff
@@ -293,7 +304,7 @@ manifest {
     description     = """call and score variants from WGS/WES of rare disease patients"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '2.0.1'
+    version         = '2.1.0dev'
```
Comment (maxulysse) [inline nextflow.config:307]:
```suggestion
    version         = '2.1.0'
```

**Review by maxulysse — APPROVED** [review]
ok for me, just missing the proper bumping version

File: assets/multiqc_config.yml (hunk 2 above)
Comment (ramprasadn) [reply assets/multiqc_config.yml]: Apparently, I had not done that 🤦🏻 But its fixed now.

File: conf/modules/align_sentieon.config [hunk 5]
```diff
@@ -17,30 +17,28 @@
...
 process {
 
-    if (params.aligner.equals("sentieon")) {
-        withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_BWAMEM' {
-            ext.args = { "-M -K 10000000 -R ${meta.read_group} " }
-            ext.prefix = { "${meta.id}.sorted" }
-        }
+    withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_BWAMEM' {
+        ext.args = { "-M -K 10000000 -R ${meta.read_group} " }
+        ext.prefix = { "${meta.id}.sorted" }
```
Comment (fellen31) [inline conf/modules/align_sentieon.config:22]:
```suggestion
        ext.prefix = { "${meta.id}_sorted" }
```

?

**Review by fellen31 — APPROVED** [review]
Looks good to me!

File: conf/modules/align_sentieon.config (hunk 5 above)
Comment (ramprasadn) [reply conf/modules/align_sentieon.config]: Good catch!

**Pull request merged**